umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -41,6 +42,137 @@
         (dissoc :Type))
     (dissoc coverage-type :Type)))
 
+(defn- duplicate-supported-projections
+  "Duplicate the SupportedProjections in v1.1 ServiceOptions to SupportedInputProjections
+  and SupportedOutputProjections in v1.2 ServiceOptions when migrate from 1.1 to 1.2."
+  [service-options]
+  (-> service-options
+      (assoc :SupportedInputProjections (seq (map #(hash-map :ProjectionName %)
+                                                  (:SupportedProjections service-options))))
+      (assoc :SupportedOutputProjections (seq (map #(hash-map :ProjectionName %)
+                                                   (:SupportedProjections service-options))))
+      (dissoc :SupportedProjections)))
+
+(defn- duplicate-supported-Formats
+  "Duplicate the SupportedFormats in v1.1 ServiceOptions to SupportedInputFormats
+  and SupportedOutputFormats in v1.2 ServiceOptions when migrate from 1.1 to 1.2."
+  [service-options]
+  (-> service-options
+      (assoc :SupportedInputFormats (:SupportedFormats service-options))
+      (assoc :SupportedOutputFormats (:SupportedFormats service-options))
+      (dissoc :SupportedFormats)))
+
+(def v1-1-supported-formats-enum->v1-2-supported-formats-enum
```

Comment:
When reading the above changes, I was curious how you would address the change from mime-type to string; nice and simple -- I like it :-)

Reply:
Thanks, @oubiwann